pkg/agents/team3/agentMain.go

```diff
@@ -62,7 +63,7 @@ func (a *CustomAgent3) Run() {
 	//receive Message
 
 	//eat
-	a.TakeFood(takeFoodCalculation(a))
+	a.TakeFood(food.FoodType(takeFoodCalculation(a)))
```

Comment:
Instead of casting here, `takeFoodCalculation()` should return a type `food.FoodType`